docs/usage.rst

```diff
@@ -51,3 +51,77 @@ without reloading page.
   <script type="{% js_type_for_cookie_consent request "social" "*:.google.com" %}" data-varname="social">
     alert("Social cookie accepted");
   </script>
+
+
+Asking users for cookie consent in templates
+--------------------------------------------
+
+``django-cookie-consent`` can website visitors a cookie consent message. This
+message informs users that the website uses cookies and requests their consent
+to store them. The script responsible for displaying the message is
+``cookiebar.js``. You can see an example of its usage in the testapp.
+ 
+In order to display the cookie consent message on your website:
+
+1. Load the ``cookiebar.js`` script in your HTML template. You can do this by
+   adding the following line to the ``<head>`` section of your template:
+
+.. code-block:: html
+  <script type="text/javascript" src={% static "cookie_consent/cookiebar.js" %}></script>
+  
+2. In your JavaScript code, call the showCookieBar function with the
+   appropriate options object:
+
+.. code-block:: javascript
+
+  showCookieBar({
```

Comment:
```suggestion
  window.showCookieBar({
```

IMO that makes it a bit more obvious that it's registered as a global

In the future, we should make these scripts properly scoped & make it possible to include it in potential bundlers like webpack/rollup etc. via a proper import system.